openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -524,26 +524,31 @@ def _get_representations(self, instance, exp_files):
         for collection in collections:
             ext = collection.tail.lstrip(".")
             preview = False
-            # if filtered aov name is found in filename, toggle it for
-            # preview video rendering
-            for app in self.aov_filter.keys():
-                if os.environ.get("AVALON_APP", "") == app:
-                    # no need to add review if `hasReviewableRepresentations`
-                    if instance.get("hasReviewableRepresentations"):
-                        break
-
-                    # iteratre all aov filters
-                    for aov in self.aov_filter[app]:
-                        if re.match(
-                            aov,
-                            list(collection)[0]
-                        ):
-                            preview = True
-                            break
-
-            # toggle preview on if multipart is on
-            if instance.get("multipartExr", False):
-                preview = True
+            # TODO 'useSequenceForReview' is temporary solution which does
+            #   not work for 100% of cases. We must be able to tell what
+            #   expected files contains more explicitly and from what
+            #   should be review made.
+            # - "review" tag is never added when is set to 'False'
+            use_sequence_for_review = instance.get(
+                "useSequenceForReview", True
+            )
+            if use_sequence_for_review:
+                # if filtered aov name is found in filename, toggle it for
+                # preview video rendering
+                for app in self.aov_filter.keys():
+                    if os.environ.get("AVALON_APP", "") == app:
+                        # iteratre all aov filters
+                        for aov in self.aov_filter[app]:
+                            if re.match(
+                                aov,
+                                list(collection)[0]
+                            ):
+                                preview = True
+                                break
+
+                # toggle preview on if multipart is on
+                if instance.get("multipartExr", False):
+                    preview = True
```

Comment:
Would this logic be more readable when moved to `use_collection_for_review` method?

Reply:
Here's what the logic could be like:

_consider as pseudocode_
```python
def use_collection_for_review(self, collection, instance):
    
    # TODO 'useSequenceForReview' is temporary solution which does
    #   not work for 100% of cases. We must be able to tell what
    #   expected files contains more explicitly and from what
    #   should be review made.
    # - "review" tag is never added when is set to 'False'
    if not instance.get("useSequenceForReview", True):
        return False
        
    # toggle preview on if multipart is on
    if instance.get("multipartExr", False):
        return True
    
    # if filtered aov name is found in filename, toggle it for
    # preview video rendering
    app = os.environ.get("AVALON_APP", "")
    aov_filters = self.aov_filter.get(app, [])
    for aov_filter in aov_filters:
        first_filename = list(collection)[0]
        if re.match(aov_filter, first_filename):
            return True
```

Reply:
The aov part have already existing PR which (I think) eradicate the remaining code into 2 lines and the `useSequenceForReview` is not proper way how to achieve the result so it's not needed. And I would say that in this case it's better that we'll see it everytime we look into the code :)

Reply:
If it's being solved already elsewhere then ignore my comment. Thanks!